### DISCUSSION TITLE
Some advice for creating a Fastify adapter ?

### DISCUSSION THREAD

**Discussion by carere** [ecyrbe/zodios#466] [Q&A]
Hello, I would like to implement a fastify adapter, does anybody have some advice for starting dev on this adapter ?

**Comment by ecyrbe** [comment]
Hello, converting to discussion.
I know someone also interrested in the topic. will ping him.

**Reply by ecyrbe** [reply]
@neolectron you may join forces.
in the mean time here's starter i did a few month ago :  
https://www.typescriptlang.org/play?&install-plugin=typescript-playground-link-shortener#code/JYWwDg9gTgLgBAbwLACg5wMIQHYwKYAeMW2AZsAOYAiepAhgK4A2MANKuvQM4zCkCe7NHABidHn34AhcXgAyEChTxQh6MRIEBJbDzrYAxnjWjxvAQGUDACzwg6Jjef4AVfmDwAFKBABuwABMVRzNJNw9vP0CVGnpmNg44AAkXF08AWTwYawgArhMAJToAdwK8MCZ+WMYWAFECMCg8Li5gHEKSsoBHBmaYavj6xubW9sSi4osVXxUZLmNxkqmoGagBlkKIBnwAcTxsFWADHXwoeiNN7bwk-QCmFUzs3Mv8CxzYa1uAeTBeHHzUABfOCkHwgOAAIm4zghAG5UKBILBEIl5lBgHQmMAAF4qLDgYD3VSJXyYwJ0GDQfFgQnBRIALVy4S8Pn8QWJKGBoIg4KhoQEAFoYO48ALGlF2QLsbk4QjwNB4MhhHQaVIGIT2SZGQE2lxatgApBgLhYsbgH9sFrcrrPBTrFwpPxHjkAladRA9QajSbaGaLY7bdkTM7nol7ABrPC1KA+KAAzkgsGQgAC0vdXAA9AZoHhZShEQrEKJCacAIIxujSfgANUxvVYcEyUGUcC5SYhqetHqzOYzWIARhntoSuAA6YUeLh5gvI7GJnmQtN51DZ3TwFVgOAAXhB-P4AAoAJTwlCoDMZuClgIBOBcGx2OhwUlYgIU6BwW63w5k3FQVAb0d5hgWsXzfKBqVpKB92fclKXAnkaSJY9-zAMBAKyZYMSxX8IKJfc0SwnE8QQyDkNPFBVx4OAVFjLhtzgCMoxjaAuH3ABtRIlXQdA9BgBguAALkhII4hYCETB4+97CE7FRwgfsACs8AMGB9y47j0BAEY6GUGTAJgdFsAoI8JNbQ8TEBIQAF0yJXf51xpeiVWANUNRUNSwyyF0hIhZQYHExIwDtHyMwC5UsXEHzbCYJgIDC9AgqgOgtNOQS4A44R0HUjTsGSvAfNyrT4o0uAJ3yyEAEVeigfhio0u9bGkuBZJ4QzjMPUcQGNfcABZzMSdBLMSKyTCaLhIF0crZPkpSVI8zK4Gi2K9Na412os-rhBolihEBQ9ElHOhr31Q0IGNVTsoYrzch8vy6rgILshCgTAnuxK8tSoSMpKlEFpyvKCry+6NLKnyqpUWrTPqqS6BWgy1qPTrur6qHW1MkaBrgMaJvmPSZuUi7MfQQI4bakyibgQqpv0snNo0vbTO2uMNoO-t1SYAIjxPVBCCReAysbFUrEauhbSSlKVC4AAeFwG1LAwDHohBAQAPnolxqKIfY8nSxJjVIFRkjwI6TFHM39cNlw6EJVArMSAB+I2js1-ADToy70CpoSLagOAADk8pd7W6NWoyTx+hqH29shDeFh9w+4wFMcd9IhZhsWPslqWKfQK3CVR9Am2UbO-pKj2fvSgBpPB+DgY1-byqyhLj+wE4rtGc+4+WDE7lWKb7hahO7xIh4V7mUF5wsBdTsAW7oaNaJluWFaV1X1aDt30p9p3XTgM3R23vOmDt4RHaSY2b0IV2dfL3j+Ojg3fYsGAKX4jedYhESan8uAAB9KYYCAfsKg27cUjk1bec9QFJwWinNOIsM4SzjCXduR8C6NhUMXTuWVsHcTYtXWu9dn6vy4E3OAUDcFDVLhpYe1C4ADx+gw7io8e7CBYePAWThJDgxqog6WiRSyOSvsHOA2pdQnW9P0X02BzRtGwGxDGwh0jvzoiGV0iRAzWBUaIrsXBNEOidNdAIUtBHAAbOkAeasdwzznogrIWdEgiBLCocsSUqwgV6Cg7iYiPQSLOj6cgMj-T8E0SYmk5iGyaJVmxbAgDgFQCsmxCE70kFTkURpBApURRg2qrVVsiRLEcJFKYTQ-AF4sS8aY7RPjPSnXOqaIJciFHBm0Wokwmjqm6P0Y6NRYSzGNksfRGxMNynIJqX4+p0jZE4ADHaPpES4BRJiXElQiSIRMzSSrcepAGCGAtHAPyXCBDajnpUoRWtN7jK9P4qRgTpnyPSY2VpRj2l2k6embphinjGNMeYlWJg57aKOfwOeqAVb7mckPcJV1vlCXSA2R61ghKaMPL9dAlF4Bpl1ICnczlRyBM5pjfc5o7Coq3GrElIBOpGO3FuHcKUXRwAAGRMrrvgKliLaU7kRYkQ8ABCBOGLmq6PPkdSW9EsUejnqOFJ9i4z23xc4qCRLKVkopey8cxS6U7ghKK9kEJeUJ3uJi3RPDa47klVwaVsrUoKvICwdyKr2VqrZXYTVHguWVVyQa4QZF0DGrgD0CGocKBfEUgTVeCc+BwH3Jas1zLWVxtyaOe4Rlsj0LgAABlRZdINNUQ1htmvAC1prk1NACAwIw+4KYQoVg2VV241a1oMGxSlo4qZWXom28BdA6ZlxgRpP1+ThBCstVIXI5rhXpmtXQcWcquB2uNIShaxLnWNtdVSgW2rITjoCLVXlfKPx0V2eGbAEBijYCPcWB1UBXGVkdB4vAXj0BXLqQEv0cjZnZHmY2SJdpomxKAaspJNrJYQkeTg0uoMd0TrzPTApyygMJKST28D490X2Sxs0eIStMb9gnTJXRu7BCYzzfwENeNw1zQpmRgtVH1x0WBWavhP74WLP-ZjPt6Bsb-HKuXAATJmzNhHp0w1HDxyaV7X2SIafcr91hn0lV+Z3eFndNH90Q-EtZEn5jgdMlQwaCcmh8SgJesa8QTwwJ5g0Qs50VDnDwDo0TIsznAHeeI65ky7kWmaYkZRwjN5tI0W8gLOsamfN6b8gZYK0VwHw3uoSTib13vcXWJ9mNpM3Nk8E0JUW2NLMA1pkDs7M5xj05jTJ0GITEYhMO9AAGVnIYhKhqyCdaPwyMnpbUhaCZSyY7kljeW-3ZBVlswKJWQBpVkt1+jUtMkk2aqOE5HWKCtjG8IHTfHMaCeE05jzb7bkfpmSEuZQ32Mjc08BiEm20OJGBKy4FozpZnaiZZ1A1m+Z11wPZugRg4AFC2PgH4Fp+HCCqaFuimWvNHYeS0iHGDvmvPTfD8LdoDGRehRYjokxpiG3hxMZYqw5iOZ3AT3HaxaDf2x90XoVF8edDwEGng6wYBDDGqMbAUsycrBUFY-7DOme3O-mzkYciudLHJ-8xYpRyiVG0RMMoFQqiU8GA0dnYvuerD5wr2XyvRKs7V6LnA4ucc86gFL4QAOrh7AOOiRW+PAd4Bt4cY432zi-ZJ-9x3zu7cnB+xcRIJB8BEBIOQVbO4g9X1D5QFnAKYa4TxxcnWwKcV7alTDVjw3rAW9ziKSIbJE-X0Y3uZk+foi+x3MC0vrJy+x8SAoJQheRHAuJw35QFeSnOFb4odvMWHdXDeAqT4BpgdyNBxpTXdIFo68F6ZHXSu5-e-2C70ykeQ84DD6ZOeCeOQaWrxKKfGk28qAKbFhlN1GwJwYFAJgyK7QJ2H3cFQQkrf4BuAaIkvTMaT939xGftOEhp8GcF9v8l9bcjhV8cBg9iAN9KAt948SIiRTJ98C9f90Bj8-xhB1srN8w3cHM08uBgVXN3NfFPN31GkcBfNLcJczd5caCidZB6If869qCZdBc6C2CACWcRcWgNd6DecmCBcuCVc6hDdeDjcf8c9+cZclcODFdKhuCxCOcTdCcBDSdgCFCRCDdhhxDOdJCTAUDy8gUS888a92R6Iq9TCD8Kd9cTAMDjDSlu9G8O8W9ZAMCYt1IfArhFN-Mk9VEXlMYOkUcuk0cekjFWMpD0BX8ndl87d5cwCXc-d3c-tScEjfc8CPdIDvt18yBKB6I18YDcjqAtCClq0FoIBfg0pojR9-hFN0BlNS5VNS5NFUYf9WihC+h2iZDKhWi0iICKYCjo8KBUYalTkXt-1UZDD2RMYmFDwhIakiCos2jpCadOiVjdcGwpjgg4AMDsD3sKJMNLVBFNxcVUIr0T0z0L0pNdEiDEgyoIBSAPxwlpdVCbCqdpdVjmctCeDlDmCtCpD59ND9cfi+DTdVgWcpDtQtjMD1tUAjjUJxNHd5pNIjFboshipr9b9IQMwXoAhipH8iQhJ9wmgugGwxoXVLpjNr9L1MkvYsMuhRwyN208p60AghISSZUJsxxAhWwE4qE9px4AIsQeB9g1IHoFQhIeoABOaUsyE8IAA
